.github/workflows/test.yml

```diff
@@ -49,6 +49,19 @@ jobs:
           path: build
           if-no-files-found: error
 
+  build-instrumented:
```

Comment:
i'm not sure what instrumented means exactly. i would vote to name this
`build-e2e` and `yarn build:e2e` so that i know that this is a different build flavor meant specifically for e2e tests. instrumented could be for production or something else